src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -535,13 +535,13 @@ well as menu structures (for main menu and popup menus).
          <shortcut value="Alt+Up" title="Move Lines Up" />
          <shortcut value="Alt+Down" title="Move Lines Down" />
          <shortcut value="Ctrl+D" title="Delete Line" if="!org.rstudio.core.client.BrowseCap.isMacintosh()" />
-         <shortcut value="Cmd+D" title="Delete Line" if="org.rstudio.core.client.BrowseCap.isMacintosh()" />
          <shortcut value="Ctrl+U" title="Yank Line Up to Cursor" />
          <shortcut value="Ctrl+K" title="Yank Line After Cursor" />
          <shortcut value="Ctrl+Y" title="Insert Yanked Text" />
          <shortcut value="Ctrl+T" title="Transpose Letters" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut value="Alt+-" title="Insert Assignment Operator"/>
          <shortcut value="Cmd+Shift+M" title="Insert Pipe Operator" />
+         <shortcut refid="quickAddNext" value="Meta+D" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
```

Comment:
I think you're right. I'll leave it unbound for now and let users generate their own bindings if desired.